### PR TITLE
bugfix/711

### LINKED ISSUE #711: Game Options not being stored correctly
Browser: Chrome
Airport: All

#### Issue Description
openscope/openscope#690 (refactor #670) appears to have caused some issues with the local storage of game options. As can currently be seen on the dev server, game options are not preserved, and are defaulted to the _first_ option in the list, rather than the default specified in `gameOptionConstants.js`.

#### Steps to reproduce
- Clear cache and load dev server
- Observe that the default values are not honored
- ~~Change some settings and refresh~~
- ~~Observe that the changes to options are not preserved~~

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #711.

Ensure game options initialize to correct default values.

### CODE REVIEW

File: src/assets/scripts/client/game/GameOptions.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default class GameOptions {
         this._options[optionProps.name] = optionProps;
         let optionValue = optionProps.defaultValue;
 
-        if (typeof storedOptionValue !== 'undefined') {
+        if (!_isNil(storedOptionValue)) {
```
Comment (n8rzz) [inline src/assets/scripts/client/game/GameOptions.js]: wait, what, really?  In the instance a key doesn't exist in `localStorage` it returns `undefined`, right?

can you add a test case for this?

Comment (erikquinn) [reply src/assets/scripts/client/game/GameOptions.js]: it's returning `null` instead of `undefined` in all cases i was seeing.

Comment (erikquinn) [reply src/assets/scripts/client/game/GameOptions.js]: so the isNil will cover both.

Comment (erikquinn) [reply src/assets/scripts/client/game/GameOptions.js]: done

**Pull request merged**